src/main/world/restaurant/customer.gd

```diff
@@ -374,9 +532,9 @@ func _update_customer_properties() -> void:
 	
 	if Engine.is_editor_hint():
 		# Skip the sound effects if we're using this as an editor tool
-		return
```

Comment:
In compliance with GDQuest's code style guidelines, I've removed the 'return' from the middle of this statement